packages/app-store/routing-forms/pages/routing-link/getServerSideProps.ts

```diff
@@ -24,9 +24,6 @@ export const getServerSideProps = async function getServerSideProps(
   const { currentOrgDomain } = orgDomainConfig(context.req);
 
   const isEmbed = params.appPages[1] === "embed";
-  if (context.query["flag.coep"] === "true") {
```

Comment:
This logic is already being handled in the middleware, https://github.com/calcom/cal.com/blob/main/apps/web/middleware.ts#L122, so we don't need this here